automation/run-tests.sh

```diff
@@ -22,9 +22,9 @@ export CLOUDSDK_CORE_DISABLE_PROMPTS=1
 curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
 
 # Installing the package
-mkdir -p /usr/local/gcloud \
-  && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
-  && /usr/local/gcloud/google-cloud-sdk/install.sh > /dev/null
+mkdir -p /usr/local/gcloud
+tar -C /usr/local/gcloud -xf /tmp/google-cloud-sdk.tar.gz
+/usr/local/gcloud/google-cloud-sdk/install.sh > /dev/null
```

Comment:
should this be strung together for "if this fails, then fail the following steps"?
or was this done because we want to enable each step independent of the previous step passing-or-failing?

Reply:
Originally this was just copy/pasted installation. This should fail if anything return an inappropriate error code. I'll add `set  -e` to the top, could of sworn it was already there. The main intent of my changes here was `-xf` option to tar, as this spams ~20k lines of noise at the start of test run logs with the verbose logging enabled.

Reply:
I made a follow-up ticket and tied it with re-enabling the azure tests for this